### PR TITLE
Update MCS_CLIENT_VERSIONS

### PR DESCRIPTION
**Pull request by bariscodefxy**

### CODE REVIEW

File: src/Protocol/ProtocolRecognizer.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 
 // Adjust these if a new protocol is added or an old one is removed:
-#define MCS_CLIENT_VERSIONS "1.8.x-1.12.x"
+#define MCS_CLIENT_VERSIONS "1.8.x-1.14.4"
 #define MCS_PROTOCOL_VERSIONS "47, 107, 108, 109, 110, 210, 315, 316, 335, 338, 340"
 #define MCS_LATEST_PROTOCOL_VERSION 340
```
Comment (bearbin) [inline src/Protocol/ProtocolRecognizer.h:12]:
```suggestion
#define MCS_PROTOCOL_VERSIONS "47, 107, 108, 109, 110, 210, 315, 316, 335, 338, 340, 393, 401, 477, 480, 485, 490, 498"
#define MCS_LATEST_PROTOCOL_VERSION 498
```

**Review by bearbin — APPROVED** [review]

**Comment by bearbin** [comment]
Looks good. On your PR you do you'll need to add yourself to the CONTRIBUTORS file, but not necessary for this small change.

**Comment by bearbin** [comment]
Oops, you should have cancelled automerge 12xx12 if you had a concern.

I think we do, we allow clients to connect so reporting that we don't doesn't make much sense.

**Comment by 12xx12** [comment]
Is that a new feature?

Your point is true. And might get more reports.

Do we advertise the versions anywhere else?

**Comment by bearbin** [comment]
Yes, github added automerge recently, I use it at work and its nice for the workflow, esp when PRs need to get merged in parallel, less manual waiting for the CI to go off.

We advertise versions on the website and readme IIRC - probably best to keep .12 there until .14+ is stable, or add a warning about experimental compatibility.

**Pull request merged**